### PR TITLE
Add Monterrey file

### PR DESCRIPTION
**Pull request by nysertxs**
this is the beginning of the monterrey file ☀️ 

please provide feedback on:
- if you've been to monterrey what you like and what you dont like
- if you've never been to monterrey what interest you about coming here
- what do you prefer hotels? airbnb? etc

@EddLeon do you know any good place to play wow in mty? 🎮

### CODE REVIEW

**Comment by EddLeon** [comment]
Yes i do, I used to play all the time @ Tec's library :+1:

**Comment by yashkmankad** [comment]
@nysertxs - Hi, your PR was below mine, I have never been to Mexico. I would definitely like to visit.

File: nysertxs-monterrey.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Monterrey, Nuevo Leon
+
+### Restaurants
+
+### Local Attractions
+
+- Cerro de la Silla 
+- Fundidora
```
Comment (nysertxs) [inline nysertxs-monterrey.md]: grate place to catch pokemons

**Comment by nysertxs** [comment]
@EddLeon added your suggestions de088e6

**Comment by readiu** [comment]
:shipit:

**Comment by nysertxs** [comment]
@yashkmankad hey i add a link to a greate place to catch pokemons in monterrey mx, maybe you wanna check it out

**Pull request merged**